aws-py-apigatewayv2-eventbridge/api/handlers.py

```diff
@@ -0,0 +1,6 @@
+import json
+
+def capture_order(event, context):
+    # For now, just log the event, including the uploaded document.
+    # That'll be enough to verify everything's working.
+    print(json.dumps({ "source": event["source"], "detail": event["detail"] }))
```

Comment:
```suggestion
    print(json.dumps({"source": event["source"], "detail": event["detail"]}))

```

Reply:
We should totally add this stuff to `make lint`.